app/workers/bgs/submit_form674_job.rb

```diff
@@ -42,14 +42,27 @@ def valid_claim_data(saved_claim_id, vet_info)
       claim.formatted_674_data(vet_info)
     end
 
-    def send_confirmation_email(user)
-      return if user.va_profile_email.blank?
+    def send_confirmation_email(user_uuid, va_profile_email, first_name)
+      return if va_profile_email.blank?
```

Comment:
Should this raise instead of silently returning?

Reply:
This is one where I copied the previous decision. My guess is that they still want to process the form even if a confirmation can't go out.

Reply:
Yeah my gut says that might be worth asking about after shipping this, maybe Steve Albers would be familiar with how this works generally with mailers and forms submissions? I.e. do we want to process the form regardless of whether we can send a mailer or not?